### PR TITLE
EMSUSD-550 - Disable Material section when Mesh type is excluded from export

### PR DESCRIPTION
**Pull request by samuelliu-adsk**
Disable the material section when user excludes Meshes from export. Currently the material is attached to the mesh so when the mesh is not exported, the material can not be exported either.

### CODE REVIEW

**Review by seando-adsk — APPROVED** [review]

**Comment by samuelliu-adsk** [comment]
Hi @seando-adsk , just realize I have some debug code forgot to remove. Added another commit

File: plugin/adsk/scripts/mayaUsdTranslatorExport.mel [hunk 1]
```diff
@@ -488,10 +489,6 @@ global proc mayaUsdTranslatorExport_SetFromOptions(string $currentOptions, int $
                 mayaUsdTranslatorExport_SetCheckbox($optionBreakDown[1], $enable, "exportDisplayColorCheckBox");
             } else if ($optionBreakDown[0] == "exportInstances") {
                 mayaUsdTranslatorExport_SetOptionMenuByBool($optionBreakDown[1], $enable, "exportInstancesPopup");
-            } else if($optionBreakDown[0] == "exportCameras"){
-                mayaUsdTranslatorExport_SetOptionMenuByBool($optionBreakDown[1], $enable, "exportInstancesPopup");
-            } else if($optionBreakDown[0] == "exportLights") {
-                mayaUsdTranslatorExport_SetOptionMenuByBool($optionBreakDown[1], $enable, "exportInstancesPopup");
```
Comment (seando-adsk) [inline plugin/adsk/scripts/mayaUsdTranslatorExport.mel:494]: Why were these lines removed?

Comment (samuelliu-adsk) [reply plugin/adsk/scripts/mayaUsdTranslatorExport.mel:494]: Those lines are some debug code I used before and forgot to remove. They are totally useless now...

Comment (seando-adsk) [reply plugin/adsk/scripts/mayaUsdTranslatorExport.mel:494]: Why would they be useless. Someone can call this function adding those strings to the input options to control the starting value of these two export options.

Comment (samuelliu-adsk) [reply plugin/adsk/scripts/mayaUsdTranslatorExport.mel:494]: Oh no those flags are not longer available. Now we use `excludeExportTypes` flag, so it will be `excludeExportTypes=[Cameras, Lights]` like this. These two flags are abandoned. It's my fault I forgot to delete them here...

**Review by seando-adsk — APPROVED** [review]
